### PR TITLE
add DataStore SSR example; fix Next.js API example; update UI imports

### PR DESCRIPTION
**Pull request by david-mcafee**
_Issue #, if available:_
https://github.com/aws-amplify/amplify-ui/issues/2256

_Description of changes:_
Add DataStore SSR example; fix Next.js API example; update UI imports

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

**Review by ErikCH — APPROVED** [review]
Thanks for catching this! Looks good

File: src/fragments/lib/ssr/js/getting-started.mdx [hunk 1]
```diff
@@ -112,3 +112,132 @@ export default function HomePage(initialData) {
...
+    );
+
+    window.location.href = `/posts/${post.id}`;
+  } catch (error) {
+    console.log(error);
+  }
+}
+
+export default function Home({ posts = [] }) {
+  console.log(posts);
```
Comment (reesscot) [inline src/fragments/lib/ssr/js/getting-started.mdx:187]: Should we have a console.log in this code example?

Comment (david-mcafee) [reply src/fragments/lib/ssr/js/getting-started.mdx:187]: Nope - will remove.

**Review by reesscot — APPROVED** [review]
Replacing the `AmplifyAuthenticator` with `Authenticator` looks good to me. Would love to see Amplify UI components being used here instead of raw html elements and CSS modules.

**Comment by david-mcafee** [comment]
> Replacing the `AmplifyAuthenticator` with `Authenticator` looks good to me. Would love to see Amplify UI components being used here instead of raw html elements and CSS modules.

+1!

**Pull request merged**